README.md

```diff
@@ -7,24 +7,26 @@
 
 This is still heavily WIP and should be treated as such. Progress can be seen in the `Projects` boards which define milestones and progress toward those milestones. When this reaches feature parity with the C API and has basic code generation in which all user facing APIs are safe, a crate will be published to `crates.io`.
 
-## Setup 
+## Setup
 
 1. Clone this repository.
 2. Run `cargo run --bin download_sdk -- -d dependencies -s 13.5.1` to download the C API dependencies.
 3. Set the `SPATIAL_LIB_DIR` environment variable to the location of the dependencies: `export SPATIAL_LIB_DIR=$(pwd)/dependencies`.
-4. Run `cargo build` 
+4. Run `cargo build`
 
 If these steps complete successfully, the `spatialos-sdk` crate has been built and linked successfully and can be used in user code.
 
 ## Running the Example Project
 
 To run the example project, you will need to:
 
-1. Run the code generator - `./spatialos-sdk/examples/project-example/codegen.sh`
-2. Build a release version of the RustWorker - `cargo build --example project-example --release`.
-3. In two terminals:
-   - Navigate to the `spatialos` directory and start spatial: `cd spatialos-sdk/examples/project-example/spatialos/ && spatial local launch`
-   - Run the example project worker - `cargo run --example project-example -- receptionist --worker_type RustWorker`
+1. Install the setup tool: `cargo install --path ./spatialos-sdk-tools --bin setup`
```

Comment:
Ah, that's a good point. I think the naming issue is ultimately moot since I plan to replace all the independent tools with `cargo spatial`, but I don't know how long it'll be before I'm ready to make that PR so I think giving the setup tool a better name is a good idea in the meantime 👍